pkg/server/home_workspaces_test.go

```diff
@@ -1245,7 +1225,7 @@ func TestCreateHomeWorkspaceRBACResources(t *testing.T) {
 					},
 				}.build()
 
-				createError := handler.createHomeWorkspaceRBACResources(testCase.context, testCase.userName, logicalcluster.New(testCase.workspaceName))
+				createError := handler.createHomeWorkspaceRBACResources(context.TODO(), testCase.userName, logicalcluster.New(testCase.workspaceName))
```

Comment:
If the test cases don't need to specify a varying context, let's use `context.Background()` here